lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -110,17 +110,58 @@ def self.create_backmerge_pr(token:, repository:, title:, head_branch:, base_bra
 
         other_action.push_to_git_remote(tags: false)
 
-        pr_body = <<~BODY
-          Merging `#{head_branch}` into `#{base_branch}`.
+        # Live playground to edit this graph
+        # https://mermaid.live/edit#pako:eNqNkU1PwzAMhv9KZanqpWVFwCVHQOKCxoFrLl7itdGaZEoTTWjqf8eLOo2i8eGTPx77TewjKK8JBJTlUbqCzTgTRTEHJ6s6E18C7vtqkc4li8Y9BnSqX6MlBqoYkttV9Y_cW9AUGLxt2_Y7Nfb-8OStNfEVNzQwtcVhpCU1XUJ2p7KU7vzAXNlkmSLQQDjS6v7m4S7nVU9q51O8UsmSv7jzSEuho9Xc3pzaG-Oib_KXlxr_QL8onbuuVy9unvrXbKiBCV645qvme0mIPVmSINjVtMU0RAm8O0YxRf_-4RQIbqca0l5jpGeDXUALIu9_-gRejKzU
+        meramid_git_graph = <<~GRAPH
+          ```mermaid
+          %%{
+            init: {
+              'gitGraph': {
+                'mainBranchName': 'trunk',
```

Comment:
Shouldn't this be `#{base_branch}` instead?

Even if most of our repos have migrated to all use `trunk` some of our repos still use a different name for their default branch anyway (`develop` for Tumblr-iOS, `master` for Tumblr-Android, `main` for some other apps from OtherBets…)

Reply:
Yes, I missed it when porting the code from the Mermaid live editor.

```suggestion
                'mainBranchName': '#{base_branch}',
```